sdk/ml/automl.tests.yml

```diff
@@ -3,4 +3,8 @@ trigger: none
 extends:
   template: ./tests.yml
   parameters:
-    TestMarkArgument: "automl_test"
\ No newline at end of file
+    TestMarkArgument: "automl_test"
+    CloudConfig:
+      Public:
```

Comment:
This and the one below would be more consistently named as `Canary` given they target euap regions.

Reply:
This is likely to changed very soon to be a prod region, so I think I'll just leave it as is